firmware/controllers/algo/engine.cpp

```diff
@@ -64,7 +64,7 @@ int MockAdcState::getMockAdcValue(int hwChannel) {
  * See also periodicFastCallback
  */
 void Engine::updateSlowSensors(DECLARE_ENGINE_PARAMETER_SIGNATURE) {
-	int rpm = rpmCalculator.rpmValue;
+	int rpm = rpmCalculator.getRpm(PASS_ENGINE_PARAMETER_SIGNATURE);
```

Comment:
All these called are not time-critical. In PROD_CODE, getRpm() actually equals to 'return rpmValue'.

Reply:
Needed by 'mockRpm' and unit-tests.